packages/yoga/src/Theme/theme/theme.js

```diff
@@ -1,6 +1,24 @@
 /* eslint-disable prefer-destructuring */
+import { merge } from '@gympass/yoga-common';
 import * as componentThemes from '../../**/*.theme.js';
 
+const getComponentThemes = tokens => {
+  const { colors, baseFont, baseFontSize } = tokens;
+  const components = {};
+
+  Object.entries(componentThemes).forEach(([names, themed]) => {
+    const [, name] = names.split('$');
+    components[name.toLowerCase()] = themed({
+      ...tokens,
+      colors,
+      baseFont,
+      baseFontSize,
+    });
+  });
```

Comment:
This is a personal opinion:

```suggestion
  const components = Object.entries(componentThemes).reduce((acc, [names, themed]) => {
    const [, name] = names.split('$');
    acc[name.toLowerCase()] = themed({
      ...tokens,
      colors,
      baseFont,
      baseFontSize,
    });
  });
```